### PR TITLE
feat: Update Releases docs with new sentry-cli set-commits behaviour

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective
Inform users of new functionality and flags available in the sentry-cli for releases.

Screen Shot 2020-07-01 at 11 10 03 AM

### CODE REVIEW

File: src/collections/_documentation/workflow/releases/index.md [hunk 1]
```diff
@@ -227,6 +227,28 @@ When Sentry sees this commit, we’ll reference the commit in the issue, and whe
...
+export SENTRY_ORG=my-org
+VERSION=$(sentry-cli releases propose-version)
+
+# Create a release
+sentry-cli releases new -p project1 -p project2 $VERSION
+
+# Associate commits with the release
+sentry-cli releases set-commits --auto $VERSION
+```
+If you do not have a repo-based integration associated with your sentry organization, then the `--auto` flag will automatically use the git tree of your local repo, and associate commits between the previous release’s commit and the current head commit with the release. If this is the first release, we’ll use the latest 20 commits. This behaviour is configurable with the `--initial-depth` flag.
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/releases/index.md:243]:
```suggestion
If you don't have a repo-based integration associated with your sentry organization, then the `--auto` flag will automatically use the git tree of your local repo, and associate commits between the previous release's commit and the current head commit with the release. If this is the first release, Sentry will use the latest 20 commits. This behavior is configurable with the `--initial-depth` flag. 
```

File: src/collections/_documentation/workflow/releases/index.md [hunk 2]
```diff
@@ -227,6 +227,28 @@ When Sentry sees this commit, we’ll reference the commit in the issue, and whe
...
+
+# Create a release
+sentry-cli releases new -p project1 -p project2 $VERSION
+
+# Associate commits with the release
+sentry-cli releases set-commits --auto $VERSION
+```
+If you do not have a repo-based integration associated with your sentry organization, then the `--auto` flag will automatically use the git tree of your local repo, and associate commits between the previous release’s commit and the current head commit with the release. If this is the first release, we’ll use the latest 20 commits. This behaviour is configurable with the `--initial-depth` flag. 
+
+Alternatively, you can use the the `--local` flag to enable this behaviour by default.
```
Comment (MimiDumpling) [inline src/collections/_documentation/workflow/releases/index.md:245]:
```suggestion
Alternatively, you can use the `--local` flag to enable this behavior by default.
```

**Comment by MimiDumpling** [comment]
I'll leave approval to @manuzope for the technical aspects

File: src/collections/_documentation/workflow/releases/index.md (hunk 1 above)
Comment (NisanthanNanthakumar) [reply src/collections/_documentation/workflow/releases/index.md:243]: lol the Canadian version `behaviour` 😄

**Comment by jonesphillip** [comment]
@NisanthanNanthakumar can we put the "Using the CLI" section above

> If you don’t want Sentry to connect to your repository, or you’re using an unsupported repository provider or VCS (e.g. Perforce), you can alternatively tell Sentry about your raw commit metadata via the API using the create release endpoint.

The section "Formatting Commit Metadata" looks to inform the use of the "create release endpoint". So I think it's confusing to have them separated by instructions on the CLI. What are your thoughts?

**Pull request merged**